TYPO3.Flow/Configuration/Settings.yaml

```diff
@@ -184,6 +184,25 @@ TYPO3:
               position: 'end'
               component: 'TYPO3\Flow\Http\Component\StandardsComplianceComponent'
 
+      trustedProxies:
+
+        # Defines Proxy servers which are trusted for setting request headers
+        # By default all proxies are trusted ('*')
+        # If you have some reverse proxies or CDN running in front of your server, you should provide
+        # an array list of those servers' addresses or address ranges (in CIDR notation) here:
+        #
```

Comment:
If you do definitely not have any reverse proxy or CDN in front of your server, you should not trust any proxies and set this option to "[]", ie. an empty list.